examples/dlp/cloud_function_example/cloud_function/main.py

```diff
@@ -0,0 +1,154 @@
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Scrubs PII from Log Exports.
+
+This Cloud Function example is intended to redact sensitive data from
+logs exported to pub/sub. It will read a pub/sub message and attempt
+to identify PII data within the jsonPayload.message field using the dlp
+api.  If any sensitive data has been found, it will rewrite the message
+with the identified data replaced with "[<INFO_TYPE>]" and post it to a
+pub/sub topic named 'scrubbed-logs'.
+
+This example only attempts to redact sensitive data within the
+jsonPaylod.message field.  If does not look at any other log field.
+"""
+
+import base64
+import copy
+import json
+import os
+
+from google.cloud import dlp_v2
+from google.cloud import pubsub_v1
+
+_INFO_TYPE_LIST = ['CREDIT_CARD_NUMBER', 'DATE_OF_BIRTH', 'EMAIL_ADDRESS']
+_OUTPUT_TOPIC_NAME = os.environ['OUTPUT_TOPIC_NAME']
+_PROJECT_ID = os.environ['GCP_PROJECT']
+
+
+def process_log_entry(event, unused_context):
+  """Entrypoint for the cloud function to redact pii from log messages."""
+
+  log_json = _read_pubsub_json(event)
+  dlp_response = _get_redacted_dlp_response(log_json['jsonPayload']['message'])
+  _post_scrubbed_message_to_pub_sub(_OUTPUT_TOPIC_NAME, dlp_response.item.value,
+                                    log_json)
+
+
+def _read_pubsub_json(event):
+  """Extracts the json payload from a pub/sub message.
+
+    Args:
+      event: A Pub/Sub event.
+
+    Returns:
+      The json_payload from a pub/sub message.
+  """
+
+  pubsub_message = base64.b64decode(event['data']).decode('utf-8')
+  json_payload = json.loads(pubsub_message)
```

Comment:
Shorten this to `return json.loads(pubsub_message)`

Reply:
Done